### PR TITLE
Optional Connected Experiences must be also enabled for Giphys to be …

### PR DESCRIPTION
**Pull request by premgan**
…allowed.

Port changes from the equivalent power-shell documentation to the UI documentation - https://docs.microsoft.com/en-us/powershell/module/skype/set-csteamsmessagingpolicy?view=skype-ps

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/messaging-policies-in-teams.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Here are the messaging policy settings that you can configure.
 <a name="bkchat"> </a>
 
 - **Chat**  Turn this setting on if you want users in your organization to be able to use the Teams app to chat with other people.
-- **Use Giphys in conversations**  If you turn this on, users can include Giphys in chat conversations with other people. Giphy is an online database and search engine that allows users to search for and share animated GIF files. Each Giphy is assigned a content rating.
+- **Use Giphys in conversations**  If you turn this on, users can include Giphys in chat conversations with other people. Note: [Optional Connected Experiences](https://docs.microsoft.com/deployoffice/privacy/manage-privacy-controls#policy-setting-for-optional-connected-experiences) must be also enabled for Giphys to be allowed. Giphy is an online database and search engine that allows users to search for and share animated GIF files. Each Giphy is assigned a content rating.
```
Comment (JohanFreelancer9) [inline Teams/messaging-policies-in-teams.md:83]:
```suggestion
- **Use Giphys in conversations**  If you turn this on, users can include Giphys in chat conversations with other people. Note: [Optional connected experiences](https://docs.microsoft.com/deployoffice/privacy/manage-privacy-controls#policy-setting-for-optional-connected-experiences) must be also enabled for Giphys to be allowed. Giphy is an online database and search engine that allows users to search for and share animated GIF files. Each Giphy is assigned a content rating.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**